ffmpeg/sign_test.go

```diff
@@ -0,0 +1,175 @@
+package ffmpeg
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"path"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/livepeer/m3u8"
+	"github.com/olekukonko/tablewriter"
+)
+
+func parseVideoProfiles(inp string) []VideoProfile {
+	profs := make([]VideoProfile, 0)
+	presets := strings.Split(inp, ",")
+	for _, v := range presets {
+		if p, ok := VideoProfileLookup[strings.TrimSpace(v)]; ok {
+			profs = append(profs, p)
+		}
+	}
+	return profs
+}
+func TranscodingWithSign(outPrefix string, bSign bool, bNvidia bool) {
+	in := "bbb/source.m3u8"
+	accel := Software
+	if bNvidia {
+		accel = Nvidia
+	}
+	dir := path.Dir(in)
+	devices := []string{"0"}
+
+	profilestring := "P720p60fps16x9,P360p30fps16x9,P720p30fps16x9"
+	profiles := parseVideoProfiles(profilestring)
+
+	f, err := os.Open(in)
+	if err != nil {
+		glog.Fatal("Couldn't open input manifest: ", err)
+	}
+	p, _, err := m3u8.DecodeFrom(bufio.NewReader(f), true)
+	if err != nil {
+		glog.Fatal("Couldn't decode input manifest: ", err)
+	}
+	pl, ok := p.(*m3u8.MediaPlaylist)
+	if !ok {
+		glog.Fatalf("Expecting media playlist in the input %s", in)
+	}
+
+	var wg sync.WaitGroup
+
+	concurrentSessions := 1
```

Comment:
Given that we're hardcoding concurrent sessions to 1, we should simplify the loop below, and do away with the use of goroutines and workgroups.

We can also remove any other un-necessary code from the benchmarking module here, like the `time.Sleep`s and extra logging and tablewriter use. Test cases usually don't need to report those details unless they fail.